src/Microsoft.AspNetCore.Razor.LanguageServer.Common/ProjectEngineFactory_1_1.cs

```diff
@@ -15,7 +15,7 @@ internal class ProjectEngineFactory_1_1 : IProjectEngineFactory
         public RazorProjectEngine Create(RazorConfiguration configuration, RazorProjectFileSystem fileSystem, Action<RazorProjectEngineBuilder> configure)
         {
             // Rewrite the assembly name into a full name just like this one, but with the name of the MVC design time assembly.
-            var assemblyName = new AssemblyName(typeof(ProjectEngineFactory_1_1).Assembly.FullName);
+            var assemblyName = new AssemblyName(typeof(RazorProjectEngine).Assembly.FullName);
```

Comment:
We do the `typeof(XYZ)` in order to derive the strong named key + version of the assembly to load. Since we're loading a Razor binary we should use its version/key instead of our own.